.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
 - '4.2'
-- 'stable'
+- 'node'
```

Comment:
Note: `stable` is a deprecated synonym for `node`.
See also https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions